spread.yaml

```diff
@@ -412,6 +419,18 @@ backends:
             - ubuntu-core-20-arm-32:
                   username: external
                   password: ubuntu
+            - ubuntu-core-22-64:
+                  username: external
+                  password: ubuntu
+            - ubuntu-core-22-32:
```

Comment:
do we need this one? I think it is not supported, we also should remove ubuntu-core-20-32

Reply:
Ah that makes sense, we don't support i386 from core20+. I'll remove them